dev-docs/bidders/eskimi.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: bidder
+title: Eskimi
+description: Prebid Eskimi Bidder Adaptor
+pbjs: true
+pbs: false
+biddercode: eskimi
+deals_supported: false
+media_types: banner
+gvl_id: 814
+sidebarType: 1
+schain_supported: true
+floors_supported: true
```

Comment:
Pleased add additional meta information found here: https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

* `media_types` : banner //  this is not needed, but makes it clearer
* `safeframes_ok`: true/false // makes the integration easier for publishers
* `ortb_blocking_supported`

Reply:
Do we need to add additional bidder parameters for ortb blocking? In the adapter code we're using prebid.js's ortb converter, but that won't populate bcat, badv and so on, right? So I guess for the initial version `ortb_blocking_supported` will be set to false, but we're going to update it later as our backend supports it fully.

Reply:
@mrobakowski sounds like a good plan to me. Merging!